app/case/views/case/docs.py

```diff
@@ -44,16 +44,15 @@ def case_detail_documents_view(request, pk):
 def sharepoint_docs_view(request, pk):
     try:
         issue = (
-            Issue.objects.check_permisisons(request).select_related("client").get(pk=pk)
+            Issue.objects.check_permissions(request).select_related("client").get(pk=pk)
         )
     except Issue.DoesNotExist:
         raise Http404()
 
-    documents, sharepoint_url, sharing_url = get_docs_info_for_case(issue, request.user)
```

Comment:
Our implementation doesn't make use of Sharing URLs, all we need to do is give write permission to the user and then display the web URL to the case folder.